tools/buildtools/Makefile

```diff
@@ -1,22 +1,12 @@
-CC := gcc
-CFLAGS := -Wall -Wextra -pedantic -std=c99 -g -O2
```

Comment:
I removed `CC :=` because it is being set a bit later again anyways.
Also changed optimization flag from `-O2` to `-Os` just because.